docs/en/glossary/glossary.asciidoc

```diff
@@ -662,8 +692,14 @@ Code is considered instrumented when it collects and reports this performance da
 //Source: Observability
 
 [[glossary-integration]] integration::
-Out-of-the-box configurations for common data sources to simplify the collection,
-parsing, and visualization of logs and metrics. Also known as a <<glossary-module,module>>.
+A bundle of assets for configuring the ingestion and visualization of data,
+or the protection of endpoints.
+Any number of integrations can be added to an <<glossary-agent-policy,{agent} policy>>.
+//Source: Observability
+
+[[glossary-integration-policy]] integration policy::
+A configuration definition for an input type.
```

Comment:
Not sure we want to use "input type" here because we also use it when we are describing standalone configs. Can we describe what the policy does rather than what it is?

Reply:
An instance of an integration that is configured for a specific use case. They are defined in an {agent policy}.